test-configs.yaml

```diff
@@ -210,6 +210,8 @@ device_types:
       # This board could boot v4.* if CONFIG_SERIAL_8250_OMAP is set. It was
       # added to multi_v7_defconfig as of 5.0.
       - blocklist: {kernel: ['v3.', 'v4.']}
+    params:
+      test_params: 'freez'
```

Comment:
The `params` used in device types should only be things that apply to the device.  The parameters you're adding are for the sleep test plan, so they should not be put here.

A better way to do that would be to create variants of the `sleep` test plan with the parameters, and then update the `test_config` entries to use those variants on the relevant device types.

```yaml
  sleep:
    rootfs: debian_buster_ramdisk
    params:
        sleep_params: standby freeze mem

  sleep_mem:
    base_name: sleep
    rootfs: debian_buster_ramdisk
    params:
        sleep_params: mem
```
Then for devices that should only be doing `mem`, picking a random one:
```yaml
  - device_type: hifive-unleashed-a00
    test_plans:
      - baseline
      - sleep_mem
```
Looking at the devices where you added `freeze`, that matches with all the test configs enabling `sleep` so `sleep_mem` isn't actually needed in practice unless we decide that some devices shouldn't be tested with other modes.

Reply:
I asked @khilman about the boards to support freeze/mem, he told me to make mem a default test and all what we have now supports freeze that's why in the sleep.sh I didn't check for a param mem only for freeze.

Reply:
Ah, that's slightly different then.  It means all the boards that don't currently run the "sleep" test plan can start running it with only "mem" (suspend to RAM) since it should be supported on all systems.  Support for "freeze" may be broken on some platforms.

So following the YAML example above, it would mean leaving all the test configs with "sleep" as they currently are, and adding "sleep_mem" to other devices to run just the "mem" test.

Reply:
I am not sure about the other boards, I tried one and it doesn't have RTC at all, what I suggest is to create a rtcwake test plan that have freeze and mem as params for the boards to run sleep test plan now and create rtcwake-mem to be used in the next future for boards just how support mem such as meson sei610.
And I am not sure about standby because we don't test it for now, do you want to add it ? and do you have any idea about the boards to support it ?

Reply:
If you add it to the test plan in this PR, it will get tested on staging.

The first version of the "sleep" plan was a very basic one, I believe we should be trying all the suspend modes and only skip those that are known to not be supported on some platforms.